src/v0/destinations/klaviyo/util.js

```diff
@@ -1,12 +1,8 @@
+const set = require('set-value');
```

Comment:
This utils has both `klaviyoV2` business utils and `batching framework utils` its better to keep them separately in to files

Reply:
done